modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -3,263 +3,388 @@ ifndef::env-site,env-github[]
 +
-. Click the plus sign to the right of the trigger:
+Because the default settings for HTTP Listener are correct, you don't need to click *Edit*.
+. Click the plus icon (+) to the right of the HTTP Listener trigger:
 +
-image::os-dc-plus-sign.png[]
+.The arrow shows the plus icon (+).
+image::os-tutorial-dc-plus-icon.png[Plus icon]
 +
-. In the *Select a component* window, search for *Obj*, then click *ObjectStore Connector*,
-and click the *Store* operation.
+. In the *Select a component* window, enter `ob` to search for the ObjectStore Connector:
 +
-image::os-dc-os-store-op.png[]
+.The arrow shows the ObjectStore Connector.
+image::os-tutorial-dc-os-search.png[Search for the Object Store Connector]
 +
-. Specify the *Key* as `#[payload.key]` and the *Value* with the `payload.value` string.
+. Click the connector, and then click the *Store* operation:
 +
-image::os-dc-store-setup.png[]
+.The arrow shows the *Store* operation.
+image::os-tutorial-dc-store-select.png[Store operation]
+. In the *Configuration* tab, specify `#[payload.key]` in the *Key* field and `payload.value` in the *Value* field:
 +
-. Click the *Run* button at the top of the Design Center window. When successful, *Running* appears.
+.The screenshot shows (*1*) the *Key* field and (*2*) the *Value* field.
+image::os-tutorial-dc-store-values.png[Key and Value fields for the Store operation]
+. Click *Test* to run the app in a temporary location.
 +
-image::os-dc-run-button.png[]
+The app starts running in a temporary environment:
 +
-. To the right of the *Run* button are three vertical dots. Click the dots, then click *Copy link*.
+.The arrow shows a message indicating the `osv2demo` app is running in a temporary environment.
+image::os-tutorial-dc-running.png[The osv2demo app running in a temporary environment]
+
+You have successfully used Design Center to configure and deploy an app with an object store.
+
+Next, you test the connection by submitting REST API commands to the Object Store API and using Runtime Manager to view the results.
+
+=== Test Your Design Center App
 
-This completes creating the first part of your application. Leave your Design Center application running for the next task where we test the application. Even if the Design Center window times
-out, the application stays running while you are logged on. Click the application in the *Projects*
-screen and look for *Status Running*.
+After you create your app in Design Center, you test your app using a `curl` command to submit a REST API command to the Object Store API and use Runtime Manager to view the results.
 
-=== Test the Design Center Application
+To test your app:
 
-. Open a command line and enter this command:
+. Click the copy icon to copy the deployment URL:
 +
-[source,console]
+.The arrow shows the deployment URL copy icon.
+image::os-tutorial-dc-copy-link.png[The copy deployment URL icon]
+. Run the following `curl` command from a command prompt to send the key-value pair. 
```

Comment:
```suggestion
. From the command line, use `curl` to send the key-value pair, replacing _DEPLOYMENT-URL_ with...: 
```
"the" key-value pair? Which? Should we remind them?
Also, moved the next sentence to join the intro to the example (which ended with a colon...which should lead directly to the example).